integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -40,6 +41,7 @@
 
 public class QueryResourceTestClient
 {
+  private final static Logger log = new Logger(QueryResourceTestClient.class);
```

Comment:
in pretty much all other places in this PR, it is called uppercase LOG . I would keep it consistent and @fjy I don't actually mind it being uppercase given that this is a static final .